server/bleep/src/lib.rs

```diff
@@ -173,7 +174,10 @@ impl Application {
         config.source.save_index_version()?;
         debug!("index version saved");
 
-        let indexes = Indexes::new(&config, sql.clone()).await?.into();
+        let indexes = Indexes::new(&config, sql.clone())
+            .await?
+            .was_index_reset(dbg!(was_index_reset))
```

Comment:
```suggestion
            .was_index_reset(was_index_reset)
```

Reply:
I think we should just move this into the constructor or create a separate constructor. We only ever initialize this once.